app/v2/models/responses/LargeMessageAuditResponse.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.responses
+
+import play.api.libs.json.Json
+import play.api.libs.json.OFormat
+import v2.models.MovementId
+import v2.models.MovementType
+
+case class LargeMessageAuditResponse(
+  movementId: MovementId,
+  movementType: MovementType,
+  xClientId: String,
```

Comment:
You could keep xClientId as Option[String] rather than passing an empty string if it's not present

Reply:
indeed, in fact I would much rather that.